pkg/types/powervs/powervs_regions.go

```diff
@@ -38,6 +38,11 @@ var Regions = map[string]Region{
 			"lon06",
 		},
 	},
+	"tor": {
+		Description: "Montreal, Canada",
+		VPCRegion:   "ca-tor",
+		Zones:       []string{"mon01"},
+	},
```

Comment:
should this be `mon` ? there is already a `tor`

Reply:
whoops, yep you're right. Fixed